src/core/utils.ts

```diff
@@ -113,7 +113,13 @@ export function reverseDeepMerge(dest, source = undefined) {
         if (isFunction(src[srcArg])) {
           dest[srcArg] = src[srcArg];
         } else {
-          dest[srcArg] = clone(src[srcArg]);
+          try {
```

Comment:
The problem that led to this pr happens with a property that is not even used on the form itself. (and some of my models are a bit complex and adding hidden fields just to set readonly  would be counterproductive).